js/webviewGestures.js

```diff
@@ -39,7 +40,8 @@ var webviewGestures = {
     return this.zoomWebviewBy(tabId, -0.2)
   },
   resetWebviewZoom: function (tabId) {
-    webviews.callAsync(tabId, 'zoomFactor', 1.0)
+    const defaultZoomLevel = parseFloat( settings.get('defaultZoomLevel') || '1' );
```

Comment:
It should be possible to store the setting value directly as a number, so we wouldn't need to parse the string.